keyboards/keebio/iris/keymaps/osiris/keymap.c

```diff
@@ -0,0 +1,140 @@
+#include "iris.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
```

Comment:
Defining the `_ADJUST` layer as `16` will increase the flash size of your firmware unnecessarily, because the compiler will insert empty layers for indexes 3-15.

Lines 7-10 should be replaced with:

```c
enum layer_names {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST
};
```

This will make and keep your layer indexes consecutive, and make it easier to change the layer definitions, simply by reordering the names, or inserting a new layer name.

Reply:
changes made to rectify this